src/reflect/scala/reflect/macros/Attachments.scala

```diff
@@ -84,6 +85,30 @@ private object Attachments {
   }
 }
 
+private[reflect] abstract class EmptyAttachments extends Attachments { self =>
```

Comment:
It looks like a `Pos` is also an empty Attachments, so is a dedicated `EmptyAttachments` still needed?

Reply:
I think so -- this way, `Position.scala` is free of much worry about attachments.